PreFreeSurfer/PreFreeSurferPipeline.sh

```diff
@@ -275,6 +280,21 @@ opts_AddOptional '--custombrain' 'CustomBrain' 'NONE_or_MASK_or_CUSTOM' "If PreF
 opts_AddOptional '--processing-mode' 'ProcessingMode' 'HCPStyleData_or__Controls_whether_the_HCP_acquisition_and_processing_guidelines_should_be_treated_as_requirements.__LegacyStyleData' "'HCPStyleData' (the default) follows the processing steps described in Glasser et al. (2013)   and requires 'HCP-Style' data acquistion.   'LegacyStyleData' allows additional processing functionality and use of some acquisitions  that do not conform to 'HCP-Style' expectations.  In this script, it allows not having a high-resolution T2w image. " "HCPStyleData"
 
 opts_AddOptional '--usejacobian' 'UseJacobian' 'TRUE or FALSE' "Whether to use jacobian modulation when correcting spin echo fieldmaps for gradient distortion" "True" # NOT IN THE ORIGNAL SCRIPT
+
+# NHP options
+opts_AddOptional '--species' 'SPECIES' 'string' "Species (default: Human)" "Human"
+opts_AddOptional '--runmode' 'RunMode' 'string' "specify from which step to resume the processing instead of starting from the beginning. Value must be one of: Default, ACPCAlignment, BrainExtraction, T2wToT1wRegAndBiasCorrection, AtlasRegistration (default: Default)" "Default"
+opts_AddOptional '--truepatientposition' 'TruePatientPosition' 'string' "True patient position (default: HFS)" "HFS"
+opts_AddOptional '--scannerpatientposition' 'ScannerPatientPosition' 'string' "Scanner patient position (default: HFS)" "HFS"
+opts_AddOptional '--betcenter' 'betcenter' 'string' "Center coordinates for BET (default: 45,55,39)" "45,55,39"
+opts_AddOptional '--betradius' 'betradius' 'int' "Radius for BET (default: 75)" "75"
+opts_AddOptional '--betfraction' 'betfraction' 'float' "Fraction for BET (default: 0.3)" "0.3"
+opts_AddOptional '--bettop2center' 'bettop2center' 'int' "Distance from top to center for BET (default: 86)" "86"
+opts_AddOptional '--brainextract' 'BrainExtract' 'string' "Brain extraction method (default: INVIVO)" "INVIVO"
+opts_AddOptional '--use-t2w-phase-zero' 'UsePhaseZero' 'TRUE/FALSE' "Indicates whether to add T2-weighted image as a phase zero volume, for bright-CSF T2w contrast acquisition types (e.g., not FLAIR)" "FALSE"
+opts_AddOptional '--bias-field-sigma-no-T2w' 'BiasFieldSmoothingSigmaNoT2w' 'value' "Bias Field Smoothing Sigma for Bias Field Correction using T1w image only (only for NHP, default: 20)" "20"
+opts_AddOptional '--betbiasfieldcor' 'BetBiasFieldCor' 'TRUE/FALSE' "Indicates whether to correct bias field for BET (default: FALSE)" "FALSE"
```

Comment:
Please use `opts_StringToBool` to sanity check this parameter for typos in both this pipeline and again in `ACPCAlignment.sh`.  It accepts yes/no, true/false, and 0/1 as input and is case insensitive, so you can just pass the 0/1 variable directly to `ACPCAlignment.sh`.